src/core/20_sql.tf

```diff
@@ -26,7 +30,7 @@ resource "azurerm_mssql_server" "this" {
   azuread_administrator {
     azuread_authentication_only = true
     login_username              = data.azurerm_key_vault_secret.administrator.value
-    object_id                   = data.azuread_user.administrator.object_id
```

Comment:
we can use reference from AzureAD with data (already defined in our terraform)

`data.azuread_group.adgroup_admin.display_name`
`data.azuread_group.adgroup_admin.object_id`